doc/generic/pgf/text-en/pgfmanual-en-library-patterns.tex

```diff
@@ -91,80 +92,81 @@ \subsection{User-Defined Patterns}
 \end{pgflibrary}
 
 \emph{Caveat:} This library is currently experimental and might change without
-notice.  There are some known shortcomings that will hopefully be fixed in the
+notice. There are some known shortcomings that will hopefully be fixed in the
 future.
 
 \begin{command}{\pgfdeclarepattern\marg{config}}
-    This command is used to declare a new pattern.  In contrast to the normal
+    This command is used to declare a new pattern. In contrast to the normal
     patterns and in the spirit of |arrows.meta| this command takes a list of
-    keys and values to define the pattern.  The following keys are available:
-    %
-\begin{key}{/pgf/patterns/name=\meta{name}}
-    The name of the pattern by which it can be used later on.
-\end{key}
+    keys and values to define the pattern. The following keys are available:
```

Comment:
Stop changing the indentation!

Reply:
Nope. That is one of the only things I can do.

We keep repeating ourselves. Bullshit happened when I did that to "thousand" files. But when only checking this very small pieces it is very very unlikely that bullshit will happen again. While reformatting I am *reading* the text, too, so it is kind of double-checked. And before committing it is checked again that no bullshit happened ...